Makefile

```diff
@@ -8,7 +8,6 @@ ifeq ($(shell uname), Darwin)
 endif
 
 all:
-	-rm $(MAKEPATH)/Dockerfile.windows
-	curl $(DOCKERFILE_SOURCE) -o $(MAKEPATH)/Dockerfile.windows
-	$(SED) -i 's/^ARG FLUENTBIT_VERSION.*/ARG FLUENTBIT_VERSION=$(VERSION)/g' $(MAKEPATH)/Dockerfile.windows
-	$(SED) -i 's/FLUENT_BIT_VERSION=.*/FLUENT_BIT_VERSION=$(VERSION)"/g' $(MAKEPATH)/.drone.yml
```

Comment:
Why was this removed?

Reply:
The last line was only in use for Linux images.